includes/api.inc

```diff
@@ -10,14 +10,24 @@
  *
  * @global stdClass $user
  *
+ * @param string $type
+ *   The type of bookmark to search for.
```

Comment:
Should probably indicate that this is optional, and the default is to return everything.